lib/src/point.dart

```diff
@@ -0,0 +1,18 @@
+part of tiled;
+
+class Point {
+  num x;
```

Comment:
num? should this be int or double?

Reply:
Yeah, thats wierd. if you look at the documentation its double in json and int in tmx:
https://doc.mapeditor.org/en/stable/reference/json-map-format/#point
https://doc.mapeditor.org/en/stable/reference/tmx-map-format/#point
So i decided to use num.

Reply:
I see. maybe just add a comment explaining then? because people who use it might need to do some casting.

Reply:
I asked the people from tiled and I misunderstood the meaning of "in pixels" because it is possible to make sub-pixel measurements. So both is double. I will fix this

Reply:
Done